src/styles/common-styles.module.scss

```diff
@@ -0,0 +1,16 @@
+.introButton {
```

Comment:
It's not clear what "introButton" is.

Reply:
Ok, I'll name it `shopNow`.
<img width="771" alt="Screenshot 2024-09-12 at 12 45 51" src="https://github.com/user-attachments/assets/51d55b26-80af-4445-ae09-e3fb3cb5c999">

Reply:
I guess it was a button variant and I suggest to call it "outlined".

Reply:
There are outline buttons without rounded corners in this template. They have different size as well.

If it's the only button with this style, put it in the header css file.